chapter2.adoc

```diff
@@ -148,10 +148,10 @@ The `sdcsr`, `sdpc` provides supervisor read/write access to the `dcsr`, `dpc`.
 | 0xaaa    | sdpc       | Supervisor debug program counter. 
 |============================================================================================
 
-The `sdcsr` is a subset of the `dcsr` formatted as shown in <<sdcsr32>> and <<sdcsr64>>, while the `sdpc` has full access to the `dpc`.
+The `sdcsr` register exposes a subset of the `dcsr`, formatted as shown in <<sdcsr32>>, while the `sdpc` provides full access to the `dpc`.
```

Comment:
Nit: I would only put "the" in front of a CSR name if the name is followed by "register".  So one could refer to "the `dcsr` register", or simply to `dcsr`.